services/user-tenant-service/src/main/java/com/sourcefuse/jarc/services/usertenantservice/UserTenantServiceApplication.java

```diff
@@ -2,12 +2,20 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.PropertySource;
 
 @SpringBootApplication
+@PropertySource({ "classpath:message.properties" })
+@ComponentScan(basePackages = {"com.sourcefuse.jarc.services.usertenantservice",
```

Comment:
Create constants for these in this file and consume it here

Reply:
sure